[EDU-3571] feat: add azion login command ref (#510)
aziontech/docs: src/content/docs/en/pages/devtools/cli/azion-cli-alpha/login/login.mdx

---

> title: Azion CLI Login

Author: ```suggestion
title: Azion CLI login
```

> description: Log in to your Azion account and save a Personal Token locally to authorize CLI commands.

Reviewer: ```suggestion
description: Log in to your Azion account and save a personal token locally to authorize CLI commands.
```

meta_tags: cli, login, authentication
namespace: documentation_cli_login
menu_namespace: cliMenuAlpha
permalink: /documentation/products/cli/login/
---

> 

Reviewer: ```suggestion

Log in to your Azion account and save a personal token locally to authorize CLI commands.

```

:::note 

> In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.

Reviewer: ```suggestion
In case one or more required fields aren't informed through the specific flags, an interactive prompt asks for the missing information.
```
